docs/area-owners.md

```diff
@@ -154,20 +154,22 @@ Note: Editing this file doesn't update the mapping used by `@msftbot` for area-s
 
 ## Operating Systems
 
-| Operating System | Lead          | Owners (area experts to tag in PRs and issues)                      | Description  |
-|------------------|---------------|---------------------------------------------------------------------|--------------|
-| os-alpine        |               |                                                                     |              |
-| os-android       | @steveisok    | @akoeplinger                                                        |              |
-| os-freebsd       |               |                                                                     |              |
-| os-mac-os-x      |               |                                                                     |              |
-| os-maccatalyst   | @steveisok    | @kotlarmilos                                                        |              |
-| os-ios           | @steveisok    | @vargaz, @kotlarmilos                                               |              |
-| os-tizen         | @gbalykov     | @hjleee, @wscho77, @clamp03, @JongHeonChoi, @t-mustafin, @viewizard |              |
-| os-tvos          | @steveisok    | @vargaz, @kotlarmilos                                               |              |
-| os-wasi          | @lewing       | @pavelsavara                                                        |              |
+`os-` labels not listed here do not have explicit ownership.
+
+| Operating System | Lead          | Owners (area experts to tag in PRs and issues)                      | Description     |
+|------------------|---------------|---------------------------------------------------------------------|-----------------|
+| os-android       | @steveisok    | @akoeplinger                                                        |                 |
+| os-freebsd       |               |                                                                     | Community owned |
```

Comment:
Does anyone know if we have community members who might want to be included here for FreeBSD like is done for tizen below?  cc @jkotas

Reply:
@wfurt and @Thefrank ?

Reply:
fine with me

Reply:
fine with me. @sec too?

Reply:
if anyone care, fine with me, I can also help on/with freebsd-arm64 as much I can/have knowledge :)

Reply:
```suggestion
| os-freebsd       |               |  @wfurt @Thefrank @sec                                              | Community owned |
```